tests/unit/processor/hyperscan_resolver/test_hyperscan_resolver_rule.py

```diff
@@ -177,7 +177,7 @@ def test_rules_with_differently_defined_but_equivalent_regex_pattern_definition_
 
 
 def test_replace_pattern_with_parenthesis_after_closing_parenthesis_not_included_in_replacement():
```

Comment:
Could you put all these test methods already in a TestClass? Then we align it a bit more to the other testcases.

Reply:
we will do it in a separate rule refactoring in gold standard